src/core/requestService/SubscriptionRequests.ts

```diff
@@ -1,7 +1,11 @@
 import MainHelper from "../../shared/helpers/MainHelper";
 import OneSignalError from "../../shared/errors/OneSignalError";
 import { logMethodCall } from "../../shared/utils/utils";
-import ExecutorResult from "../executors/ExecutorResult";
+import ExecutorResult, {
```

Comment:
unneeded import

Reply:
Import is need in this file, but just removed the 'default' import here.